examples/gnrc_networking_subnets/README.md

```diff
@@ -39,7 +39,7 @@ Start more `native` instances to simulate a cascading tree network:
 
 It is also possible to connect non-routing leaf nodes with a single interface:
 
-    make term LEAF=1 PORT=tap_b2
+    make all term LEAF=1 PORT=tap_b2
```

Comment:
Better go with 
```suggestion
    make flash term LEAF=1 PORT=tap_b2
```

this way `term` is executed after the build, even with parallel make ;-).

Reply:
`make flash` on `native` feels weird